examples/Application/Traffic-Light-Control/README.md

```diff
@@ -0,0 +1,87 @@
+
+And results of the last two rows of the table ,`presslight*` and `FRAP*`, they are the results of the code [tlc-baselines](https://github.com/gjzheng93/tlc-baselines) provided from the paper authors' team. We run the [code](https://github.com/gjzheng93/tlc-baselines) just changing the yellow signal time and the action intervals to keep them same as our config as the papers without changing any other parameters. `--` in the table means the origins code doesn't perform well in the last four `anon_4X4` datas, the average travel time results of it will be more than 1000, maybe it will perform better than the `max_pressure`if you modify the other hyperparameters of the DQN agents, such as the buffer size, update_model_freq, the gamma or others.
+
+## How to use
+### Dependencies
++ [parl>=1.4.3](https://github.com/PaddlePaddle/PARL)
++ torch==1.8.1+cu102
++ cityflow==0.1
+
+### Training 
+First, download the data from [here](https://traffic-signal-control.github.io/) or [MPLight data](https://github.com/Chacha-Chen/MPLight/tree/master/data) and put them in the `data` directory. And run the training script. The `train_presslight.py `for the presslight, each intersection has its own model as default(you can also choose to train with that all the intersections share one model in the script, just as what the paper MPLight used, it is suggested when the number of the intersections is large, just setting the `--is_share_model` to `True`).
+```bash
+python train_presslight.py --is_share_model False
+```
+
+If you want the train the `FRAR`, you can run the script below:
+```bash
+python train_FRAP.py 
+```
+
+If you want to compare the different results, you can load the right model path in the `config.py` and the right data path in the `config.json`, and then run:
+```bash
+python test.py 
+```
+
+### Contents
++ agent 
+    + `agent.py`
+    The agent that uses the PARL agent mode, it will be used when training the RL methods such as `presslight` or `FRAP` and so on.
+    + `max_pressure_agent.py` and `sotl_agnet.py`.The classic methods of the TLC. 
++ data
+    + You can get the data of the from here or download other data and put them here.
++ example
+    + Put the `config.json` here, need to change the path of the roadnet the flow data in the `json` file.
++ model
+    + Different algorithms have different models.
++ obs_reward
+    + Different algorithms have different obs and rewards generators.
+
+
+### Something about the Distributed Training
+
+We don't use the distributed traing or the parallel actors for collect the datas from the cityflow env, if you want to use the parallel actors with the cluster, you can refer to [here](https://github.com/PaddlePaddle/PARL/tree/develop/examples/A2C) or our [documentation](https://parl.readthedocs.io/en/latest/parallel_training/setup.html) for details. 
+
+### Some Suggestions and Conclusions
```

Comment:
Please remove the section. PARL will not provide suggestions for choosing the algorithm.

Reply:
fixed.